gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Thu Feb 02 14:15:44 PST 2017
+#Wed Jan 10 12:12:46 CET 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-all.zip
```

Comment:
Please, keep `-all` suffix. it is useful for debugging.

Reply:
Please make sure that @michalkurka agrees with this change.

Reply:
one more comment: do not forget to call `./gradlew wrapper` and commit re-generated gradle artefacts.

Reply:
Sure, this is more of an experimental PR

Reply:
> one more comment: do not forget to call ./gradlew wrapper and commit re-generated gradle artefacts.

I think it should be there

Reply:
If I'm correct, than the version 3.3 does not support Java 9 at all. It was the main reason why I tried upgrade in the first place